### DISCUSSION TITLE
Coding style and dev configs

### DISCUSSION THREAD

**Discussion by ehsanmim** [opensheetorg/opensheet#1] [General]
Here we want to choose our coding style and reason a config we add to the project.

**Comment by ehsanmim** [comment]
This seems to be a good starting point:
https://www.luckymedia.dev/blog/laravel-breeze-with-inertia-react-eslint-prettier-pint-and-husky

**Reply by SudoFrost** [reply]
I think this is a great idea, but we should make some adjustments.
